src/gitHubIssueFetcher.ts

```diff
@@ -37,6 +37,8 @@ export interface GitHubIssue {
     createdAt: string;
     repository: GitHubRepository;
     url: string;
+    updatedAt: string;
+    labels: { edges: Array<{ node: { name: string}}> };
```

Comment:
We could create an interface (or type) named `GitHubLabelEdge` which could be `{ node: { name: string}}`. That would avoid a bit of duplication (lines 41 and 68), as well as giving this odd-looking string of characters a meaningful name.

Reply:
Fair. I've updated this